src/js/components/Home/Countries.js

```diff
@@ -1,54 +1,60 @@
 import React, {Component} from "react";
 import Map from "./Map";
-import './MapStyles.sass'
 import ReactFlagsSelect from 'react-flags-select';
+import  countries from 'react-flags-select/es/countries';
+import Geocode from "react-geocode";
 
+Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
 
 class Mapbuttons extends Component {
     constructor() {
         super()
-        this.state = {center: [0, 0]}
-    }
-    changeCenter = center => () => {
-        this.setState({center})
+        this.state = {
+            center: [0, 0]
+        }
+        this.getCountryCode = this.getCountryCode.bind(this);
+        this.setCountry = this.setCountry.bind(this);
     }
 
-     onSelectFlag=countryCode=>() =>{
-        console.log(countryCode);
-        return this.changeCenter([-122.4194, 37.7749])
+    setCountry(countryCode) {
+        this.getCountryCode( countries[countryCode]);
     }
 
+    getCountryCode(country) {
+        Geocode.fromAddress(country).then(
+            response => {
+                const { lat, lng } = response.results[0].geometry.location;
+                this.setState({center: [lng,lat] });
+            },
+            error => {
+                console.error(error);
+            }
+        );
+    }
 
     render() {
         return (
             <div className="container-fluid">
                 <div className="row">
-                    <div className="col-9">
+                    <div className="col-md-9 col-12">
                         <Map center={this.state.center}/>
                     </div>
-                    <div className="col-3">
+                    <div className="col-md-2 col-12">
+                        {/*DropDown option where country is selected, with configuration */}
                         <ReactFlagsSelect  searchable={true}
-                                           defaultCountry="US"
+                                           defaultCountry= "South Africa"
```

Comment:
Should this not be "ZA"?

Reply:
thank you I did update to ZA you are 100% correct.